MeesGame/Gameplay/GameObjects/Player.cs

```diff
@@ -129,6 +133,31 @@ public bool HasKey()
             }
             return false;
         }
+
+        public void walkAnimation(string spritesheet)
+        {
+            Stopwatch time = new Stopwatch();
+            time.Start();
+            
+            Thread animationThread = new Thread(() => {
```

Comment:
I wouldn't know how to do this without a thread, simply removing the thread removes the animation. I don't think using a Thread here is a problem though, since the threads always stop very quickly, but if you know a better way I'll change it.